app/helpers/form_helper.rb

```diff
@@ -202,17 +202,27 @@ def file_field_f(f, attr, options = {})
   end
 
   def autocomplete_f(f, attr, options = {})
-    field(f, attr, options) do
-      path = options.delete(:path) || send("#{f.object.class.pluralize.underscore}_path") if options[:full_path].nil?
-      auto_complete_search(attr,
-                           f.object.send(attr).try(:squeeze, " "),
-                           options.merge(
-                             :placeholder => _("Filter") + ' ...',
-                             :path        => path,
-                             :name        => "#{f.object_name}[#{attr}]"
-                           )
-      ).html_safe
-    end
+    props = {
+      name: "#{f.object_name}[#{attr}]",
+      controller: options[:path] || auto_complete_controller_name,
+      url: options[:path] || '',
+      isDisabled: options[:disabled] || false,
+      query: f.object.search || '',
+      error: f.object.errors[attr] && f.object.errors[attr][0],
+      useKeyShortcuts: options[:use_key_shortcuts] || false,
+      id: options[:id] || "autocomplete-#{Foreman.uuid}"
+    }
+
+    container_class_name = "continaer-#{props[:id]}"
+    content_tag(:div, nil, :class => container_class_name) +
+      mount_react_component(
+        'ComponentWrapper',
```

Comment:
`ComponentWrapper` is not needed anymore, use `fltattenData` flag instead

Reply:
Thanks, can we remove `ComponentWrapper` from our code if we don't want to use it anymore?